alonzo/test/lib/Test/Cardano/Ledger/Alonzo/AlonzoEraGen.hs

```diff
@@ -401,6 +421,14 @@ instance Mock c => EraGen (AlonzoEra c) where
           then pure txns
           else myDiscard "TotExUnits violation: genEraTweakBlock: AlonzoEraGem.hs"
 
+  hasFailedScripts = (== IsValid False) . (getField @"isValid")
+
+  amountToFees tx (UTxO utxo) =
```

Comment:
It's a tough one. I don't want anyone to confuse it with the fee calculation. It's either the fee amount listed in the body, or the sum of the collateral. :thinking:

Reply:
I know, I was also trying to think of a name that doesn't include a "fee". Maybe `txAmountCollected` ?

Reply:
how about `amountToFeePot`?

Reply:
It doesn't matter that much, if the ones I suggested don't sound appealing then feel free to ignore this suggestion altogether.  I only commented because I had to look at the implementation to see what that function means.

Reply:
`feeOrCollateral`?

Reply:
It ain't pretty but conveys the intent very well.

Reply:
I'd be happy with that for my epitaph :)